imagetest/test_suites/image_boot/setup.go

```diff
@@ -43,12 +49,12 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 		return nil
 	}
 
-	vm3, err := t.CreateTestVM("vm3")
+	vm4, err := t.CreateTestVM("vm4")
 	if err != nil {
 		return err
 	}
-	vm3.AddMetadata("start-time", strconv.Itoa(time.Now().Second()))
-	vm3.EnableSecureBoot()
-	vm3.RunTests("TestGuestSecureBoot|TestBootTime")
+	vm4.AddMetadata("start-time", strconv.Itoa(time.Now().Second()))
+	vm4.EnableSecureBoot()
```

Comment:
does this test need secure boot?

Reply:
Well adding secure boot does not affect the boot time